Screen-Sharing-Video-Chat /js/app.js

```diff
@@ -0,0 +1,122 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
+let sessionId;
+let token;
+let screenPublisher;
+let session;
+let publisher;
+let canPublish = false;
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+
+function toggleScreen() {
+  // If the screen publisher is connected to the session, unpublish screen and re-publish camera footage
+  if (screenPublisher.session) {
+    session.unpublish(screenPublisher);
+    session.publish(publisher, handleError);
+    document.getElementById('publishScreen').classList.remove('toggle-button-on');
+    document.getElementById('publishScreen').classList.add('toggle-button-off');
+  // Else, if the camera publisher is connected to the session, unpublish camera footage and re-publish screen
+  } else if (publisher.session) {
+    session.unpublish(publisher);
+    if (canPublish) {
+      session.publish(screenPublisher, handleError);
+    }
+    document.getElementById('publishScreen').classList.add('toggle-button-on');
+    document.getElementById('publishScreen').classList.remove('toggle-button-off');
+  }
+}
+
+function preventDefaults() {
+  // If a stream is destroyed BY THIS CLIENT's publishers override default behaviour.
+  publisher.on('streamDestroyed', function (event) {
```

Comment:
Lint error: `warning  Unexpected unnamed function` (here on line 38 and in lines 44, 55, 70, and 89).